app/actions/EventActions.js

```diff
@@ -386,13 +386,16 @@ export function unfollow(followId: number, eventId: number): Thunk<Promise<*>> {
     );
 }
 
-export function isUserFollowing(eventId: number, userId: number) {
-  return callAPI({
-    types: Event.IS_USER_FOLLOWING,
-    endpoint: `/followers-event/?target=${eventId}&follower=${userId}`,
-    method: 'GET',
-    meta: {
-      errorMessage: 'Henting av hendelse følgere feilet'
-    }
-  });
+export function isUserFollowing(eventId: number): Thunk<Promise<*>> {
+  return (dispatch, getState) =>
```

Comment:
Guess this doesn't need to be a Thunk any more?

Reply:
Nope, just didn't change everything after deciding to try to fix this in backend instead. Didn't have that much free time during hælga :upside_down_face: